lib/driverProviders/browserStack.ts

```diff
@@ -43,6 +39,35 @@ export class BrowserStack extends DriverProvider {
           hostname: 'www.browserstack.com',
           port: 443,
           path: '/automate/sessions/' + session.getId() + '.json',
+          method: 'GET',
+          headers: headers
+        };
+
+        var req = https.request(options, (res) => {
+          res.on('data', (data: Buffer) => {
+            var info = JSON.parse(data.toString());
+            if (info && info.automation_session && info.automation_session.browser_url){
+              logger.info(
+                'BrowserStack results available at ' +
+                info.automation_session.browser_url);
+            } else {
+              logger.info(
+                'BrowserStack results available at ' +
+                'https://www.browserstack.com/automate');
+            }
+          });
+        });
+        req.end();
+        req.on('error', (e: Error) => {
+          logger.info(
+            'BrowserStack results available at ' +
+            'https://www.browserstack.com/automate');
+        });
+        var jobStatus = update.passed ? 'completed' : 'error';
+        options = {
```

Comment:
Nit: Instead of redefining `options`, the only thing we are changing is options.method. Maybe switch this to `options.method = 'PUT';`